proposals/0448-type-variable-scoping.rst

```diff
@@ -331,8 +415,12 @@ Effects
 Alternatives
 ~~~~~~~~~~~~
 
-1. Previous versions of this proposal, along with the accepted `#285`_, use ``-XNoPatternSignatureBinds`` instead of ``-Wpattern-signature-binds``.
-   However, there seems to be no good reason this must be an extension, instead of a warning.
+1. We could further break down ``-XImplicitBinds``, like before.
+
+   But fixing the drafting error would require a *third* extension, ``-XNegativeSignatureBinds``, in addition to the original two.
```

Comment:
I think that you are unambiguously allowed to propose modification to unimplemented proposal.

Reply:
My point wasn't that I felt that I couldn't change it, but that I didn't want to be responsible for for *even more* language extensions. 448 already adds a lot already, and I didn't want to in modifying the proposal give credence to that objection to 448.

Reply:
This is not what you're saying though. As it currently stands, I think it's rather misleading in fact.

Reply:
Ah I now gather that "But fixing the drafting error" is what is confusing. See https://github.com/ghc-proposals/ghc-proposals/pull/532/files#r1038961339 where I tried to reword this.